qlib/kernel/arch/aarch64/mm/pagetable.rs

```diff
@@ -119,9 +120,13 @@ bitflags! {
     pub struct PageTableFlags: u64 {
         const VALID           = 1;
```

Comment:
could you please add the other flags: PXN , ....
would be nice if they followed the manual for the sake of completeness.

if these flags are exclusive to aarch64 and not shared with x86 can we prefix with PT_?

Reply:
> if these flags are exclusive to aarch64, and not shared with x86, could we prefix them with PT_?

Refactoring those names would be beyond the scope of this PR. We can resolve this in another issue.